CRM/Case/Form/Activity/ChangeCaseStatus.php

```diff
@@ -109,6 +109,7 @@ public static function buildQuickForm(&$form) {
           $valdefault,
           FALSE
         );
+        $form->_caseStatus[$valdefault] = CRM_Core_PseudoConstant::getLabel('CRM_Case_BAO_Case', 'case_status', $valdefault);
```

Comment:
@mattwire what is the purpose of this?

Reply:
It makes sense now I've removed the line above...